samples/boot/webflux-x509/src/main/java/sample/WebfluxX509Application.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sample;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.web.server.ServerHttpSecurity;
+import org.springframework.security.web.server.SecurityWebFilterChain;
+import reactor.core.publisher.Mono;
+
+@SpringBootApplication
+public class WebfluxX509Application {
+
+	@Bean
+	public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
+		return http
```

Comment:
Please fix this [indentation](https://spring.io/blog/2013/07/11/spring-security-java-config-preview-readability/)

```java
// @formatter:off
http
    .x509()
        .authenticationManager(Mono::just)
	    .and()
	.authorizeExchange()
        .anyExchange().permitAll();
// @formatter:on
return http.build();
```